pl_examples/hydra_examples/conf/arch/basic.yaml

```diff
@@ -1,4 +1,4 @@
-# @package _group_
+# @package model
```

Comment:
Why the inconsistency between the package and the directory?

Reply:
yeah, let's try to maintain the convention that the config group matches the config package.